cli/tests/conftest.py

```diff
@@ -131,7 +131,8 @@ def _boto3_stubber(service, mocked_requests):
 
     # Assert that all mocked requests were consumed and deactivate all stubbers.
     for stubber in created_stubbers:
-        stubber.assert_no_pending_responses()
+        if "noassertnopendingresponses" not in request.keywords:
```

Comment:
can we avoid doing this by not stubbing requests that are not needed?

Reply:
Done. I changed the code to not call `pcluser_config.validate()`, and only validate sections and params to be tested